pkg/member/member_control.go

```diff
@@ -299,3 +303,30 @@ func (m *memberControl) IsLearnerPresent(ctx context.Context) (bool, error) {
 
 	return miscellaneous.CheckIfLearnerPresent(learnerCtx, cli)
 }
+
+// IsScaleup determines whether a etcd cluster is getting scale-up or not.
+func IsScaleup(ctx context.Context, m Control) (bool, error) {
+	logger := logrus.New().WithField("actor", "scale-up")
+	// fetch pod name from env
+	podNS := os.Getenv("POD_NAMESPACE")
```

Comment:
make `podNamespace` as an additional field in `memberControl` struct and access that.